tests/test_omegaconf.py

```diff
@@ -328,6 +328,25 @@ def test_is_none(fac: Any, is_none: bool) -> None:
     assert OmegaConf.is_none(cfg, "node") == is_none
 
 
+@pytest.mark.parametrize(
+    ("cfg", "key", "is_none"),
+    [
+        ({"foo": "${none}", "none": None}, "foo", True),
+        ({"foo": "${missing_node}"}, "foo", False),
+        ({"foo": "${missing_resolver:}"}, "foo", False),
+        ({"foo": "${oc.decode:'[1, '}"}, "foo", False),
+    ],
+)
+def test_is_none_interpolation(cfg: Any, key: str, is_none: bool) -> None:
+    cfg = OmegaConf.create(cfg)
+    assert OmegaConf.is_none(cfg, key) == is_none
+
+
+def test_is_none_invalid_node() -> None:
+    cfg = OmegaConf.create({})
+    assert OmegaConf.is_none(cfg, "invalid")
```

Comment:
I would say this one is actually a bug.

Reply:
Ok -- not going to fix it though since it's being deprecated